coveo-testing/coveo_testing/mocks.py

```diff
@@ -340,78 +340,27 @@ def ref(
     Returns a tuple meant to be unpacked into the `mock.patch` or `mock.patch.object` functions in order to enable
     refactorable mocks.
 
-    In order to target the module where the mock will be used, use the `context` argument. It can be either:
-        - A module name as a string (e.g.: "coveo_testing.logging", but more importantly, __name__)
-        - A symbol that belongs to the module to patch (i.e.: any function or class defined in that module)
-        - An instance, when patching special functions with `obj=True`
-
+    The idea is to provide the thing to mock as the target, and sometimes, the thing that is being tested
+    as the context. Refer to `coveo-testing`'s readme to better understand when a context is necessary.
 
-    The "tl;dr" is to provide the thing to mock as the target, and the thing that is being tested as the context.
     For instance, pass the `HTTPResponse` class as the target and the `my_module.function_to_test` function

```

Comment:
```suggestion
    For example, pass the `HTTPResponse` class as the target and the `my_module.function_to_test` function
```

When I read this at first, I though you were talking about mocking instances and I got really confused. I think that switching to example here will avoid this confusion.